runtime/src/assets/stylesheets/shared/_custom-mixins.less

```diff
@@ -5,9 +5,13 @@
 // functions
 /* CSS3 calc() function to perform calculations */
 /* this is temporary until Less conversion is completed */
-.calc(@property; @val) {
-  @{property}: calc(~'@{val}');
+.calc(@prop; @val) {
```

Comment:
What is the function for? Calculations work in Less. This seems unnecessarily complicated.